### ISSUE TITLE
Accessibility Issue: The screenreader doesn't announce the Required option state

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#3651]
[Example](https://plnkr.co/edit/nerje3PuT6PNz4JX).
The screenreader should announce the Required field state change: enabled/disabled
![image](https://user-images.githubusercontent.com/22372972/200818719-fc3525e4-c3c8-438f-8a6a-e1942b270511.png)

Original issue: [T11203 - Accessibility issue: Screenreader Narrator is not announcing the "Required" button status active or inactive](https://surveyjs.answerdesk.io/internal/ticket/details/T11203).

**Comment by dmitrykurmanov** [comment]
https://github.com/surveyjs/survey-library/commit/38e7259c681de7ce9469bdfdcdbb6b9969f3ec9e

**Comment by JaneSjs** [comment]
The NVDA reader doesn't announce the Required state: enabled/disabled.
